docs/monitoring.md

```diff
@@ -350,6 +350,7 @@ set of sinks to which metrics are reported. The following instances are currentl
 * `worker`: A Spark standalone worker process.
 * `executor`: A Spark executor.
 * `driver`: The Spark driver process (the process in which your SparkContext is created).
+* `shuffleService`: The Spark shuffle service
```

Comment:
I'm adding a period at the end myself on merge so that we don't get a followup patch to do it later 😄